install.sh

```diff
@@ -150,11 +150,11 @@ else
 
   # rm -rf "${THEME_SRC_DIR}/sass/_gtk-base-temp.scss"
 
-  if (is_my_distro "arch" || is_my_distro "void") && has_command xfce4-session; then
-    msg="XFCE: you may need to logout after changing your theme to fix your panel opacity."
-  elif (is_my_distro "solus") && has_command gnome-shell; then
+  if (is_running "xfce4-session"); then
+    msg="XFCE: you may need to run 'xfce4-panel -r' after changing your theme to fix your panel opacity."
+  elif (is_running "gnome-session") && [[ "${GNOME_VERSION}" == "new" ]]; then
```

Comment:
```patch
- elif (is_my_distro "solus") && has_command gnome-shell; then
+ elif (is_running "gnome-session") && [[ "${GNOME_VERSION}" == "new" ]]; then
```
Please explain this line

Reply:
You can easily do a google search for information Solus is starting to use GNOME 40

Reply:
I think solus has its own creations different from Gnome 40, you should revert it